README.md

```diff
@@ -14,148 +14,80 @@ Metro Board Reports is a member of the [Councilmatic family](https://www.council
 
 ## Setup
```

Comment:
Let's add a quick note about the Wiki too—I don't think it's obvious that we have used the wiki function.

Reply:
this feels out of scope for this pr.